docker-compose.yml.orig

```diff
@@ -0,0 +1,53 @@
+version: '2'
+services:
+  cami-store:
+    build:
+      context: .
+      dockerfile: docker/cami-store/Dockerfile
+    ports:
+<<<<<<< Updated upstream
```

Comment:
@iprunache I think this file shouldn't be in the repo. It's a merge artifact from git. Thanks!